daml_dit_if/main/jwt.py

```diff
@@ -11,12 +11,14 @@
 from .log import LOG
 
 
-
 DEFAULT_POLL_INTERVAL = timedelta(seconds=10)
 IAT_SKEW = timedelta(seconds=15)
 MAX_TOKEN_EXPIRY = timedelta(days=1)
 
 
+JWTClaims = Mapping[str, Any]
```

Comment:
🎉 types ftw

Reply:
now if only there were some corresponding types to expand within the value position here 🙂